corehq/apps/cloudcare/templates/cloudcare/partials/form_entry_templates.html

```diff
@@ -163,20 +163,19 @@ <h1 class="title" data-bind="text: title, visible: !showInFormNavigation()"></h1
           </div>
         </div>
       {% endif %}
-      <div id="submit-button" class="form-actions form-group noprint-sub-container" data-bind="visible: showSubmitButton">
-        <div data-bind="
-          css: submitClass,
-          style: {
-            'bottom':isAnchoredSubmitStyle && {{ request.couch_user.can_edit_data|yesno:'true,false' }} {% if environment != "web-apps" %} && false {% endif %} ? '30px' : '' {# data preview bar #}
-          }">
+      <div id="submit-button" class="form-actions form-group noprint-sub-container"
+           data-bind="visible: showSubmitButton,
+                      css: { 'sticky-submit': isAnchoredSubmitStyle },
+                      style: {
+                        'bottom':isAnchoredSubmitStyle && {{ request.couch_user.can_edit_data|yesno:'true,false' }} {% if environment != "web-apps" %} && false {% endif %} ? '30px' : '0' {# data preview bar #}
```

Comment:
This bottom margin was added so that the anchored submit bar sits vertically above the "data preview" bar (and all those conditionals is to check that the data preview bar is visible). So this should be able to be removed now.